docs/introduction/domain-settings.md

```diff
@@ -0,0 +1,140 @@
+
+#### 2.2 Set up the url address
+Set the url address for the domain in `app/config/domains_urls.yml`.
+
+*Note: When you add a domain with the new url address on the MacOS platform, you need to enable this url address also in the network interface, see [Installation Using Docker for MacOS](https://github.com/shopsys/shopsys/blob/master/docs/installation/installation-using-docker-macos.md#11-enable-second-domain-optional)*
+
+#### 2.3 Set up the application as "multidomain" 
+Modify the value of the parameter `is-multidomain` in `build.xml` to `true` (this is the default value).
+This parameter modifies the behaviour of some phing targets - for example, the import of the demo data is modified in such a way, that the application will try to import demo data for another domain if they are available.
+
+#### 2.4 Locale settings
+Set up the locale of the domain according to the instructions in the section [Locale settings](#3-locale-settings)
+
+#### 2.5 Generate assets for the new domain
+In order to properly display the new domain, assets need to be generated
+```
+php phing grunt
+```
+
+### 2.6. Create elasticsearch definition for the new domain
+The configuration for elasticsearch must be created for each domain in a separate json file.
+By default, the configurations for the domain 1 and 2 are parts of a microservice.
+If you add a new domain, you need to create an elasticsearch configuration for this new domain in your custom directory.
+A configuration example for elasticsearch can be found in [Definitions of Microservice Product Search Export](https://github.com/shopsys/shopsys/tree/master/microservices/product-search-export/src/Resources/definition).
+For this action follow the instructions in [How to change the default index, data export setting, and searching behaviour](https://github.com/shopsys/shopsys/blob/master/docs/introduction/product-search-via-elasticsearch.md#how-to-change-the-default-index-data-export-setting-and-searching-behaviour).
+
+### 3. Locale settings
+Some parts of this instructions are already prepared for the locales en and cs.
+
+#### 3.1 Set up the locale for domain
+Set up the locale of the domain in `app/config/domains.yml`.
+This configuration file contains pieces of information about the domain ID, the domain identifier for the domain bookmarks in the administration, the domain locale.
+
+#### 3.2 Frontend routes
+Create a file with the frontend routes for the added locale if this file is not already created for this locale.
+Create this file in the directory `ShopBundle/Resources/config/` with the name `routing_front_xx.yml` where `xx` replace for the code of added locale.
+Import this new routes configuration in `app/config/packages/shopsys_shop.yml`
+
+#### 3.3 Translations and messages
+In order to correctly display the labels like *Registration*, *Cart*, ..., create a file with translations of messages in `ShopBundle/Resources/translations/`.
```

Comment:
could you please provide a command that I should use for that?
Either 
```
php phing dump-translations
```
or something like
```
cp ShopBundle/Resources/translations/messages.en.po ShopBundle/Resources/translations/messages.xx.po
cp ShopBundle/Resources/translations/validators.en.po ShopBundle/Resources/translations/validators.xx.po
```